remote-trait-object-tests/src/lib.rs

```diff
@@ -22,6 +22,6 @@ extern crate remote_trait_object_macro as rto_macro;
 mod test_concurrent_ping;
```

Comment:
In the commit: "Rename ipc to transport"

Could you add a simple description of why "transport" is a better name than "ipc"?

Reply:
done